elements/lisk-chain/test/unit/process.spec.ts

```diff
@@ -863,6 +863,13 @@ describe('blocks/header', () => {
 			it('should not call account update', async () => {
 				expect(storageStub.entities.Account.upsert).not.toHaveBeenCalled();
 			});
+
+			it('should not update burnt fee on chain state', async () => {
```

Comment:
Seems this is related to the genesis block so in that case it would be nice to include it as part of the description of the case.

Reply:
the outer scope is `applyGenesis` so it should be clear enough